x/qgb/keeper/query_valset_confirm.go

```diff
@@ -0,0 +1,14 @@
+package keeper
+
+import (
+	"context"
+	"github.com/celestiaorg/celestia-app/x/qgb/types"
+)
+
+// ValsetConfirm queries the ValsetConfirm of the gravity module
```

Comment:
here's another instance of 'gravity' that we should eventually replace with qbg

Reply:
my bad, will fix